src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -458,11 +458,11 @@ trait WorkspaceApiService extends HttpService with PerRequestCreator {
     new ApiResponse(code = 500, message = "Rawls Internal Error")
   ))
   def copyMethodConfigurationRoute = cookie("iPlanetDirectoryPro") { securityTokenCookie =>
-    path("workspaces" / Segment / Segment / "methodconfigs" / "copy" ) { (workspaceNamespace, workspaceName) =>
+    path("workspaces" / Segment / Segment / "methodconfigs" / Segment / Segment / "copy" ) { (workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigurationName) =>
       post {
         entity(as[MethodConfigurationName]) { srcMethodConfigurationName =>
           requestContext => perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor),
-            WorkspaceService.CopyMethodConfiguration(workspaceNamespace, workspaceName, srcMethodConfigurationName))
+            WorkspaceService.CopyMethodConfiguration(MethodConfigurationName(methodConfigurationNamespace,methodConfigurationName,WorkspaceName(workspaceNamespace, workspaceName)), srcMethodConfigurationName))
```

Comment:
Clone workspace does `/<source workspace>/clone`, whereas here we're doing `<dest workspace>/methodconfigs/<dest mconf>/copy`.

We should come to a decision on whether we use "copy" or "clone" as the name of this operation, and be consistent. My vote is for "copy".

We should also be consistent about whether the source or the destination lives in the URL (and the other goes into the POST payload). My instinct is that "copy from this source" makes more sense than "copy into this destination". But the case where we're copying from the method repo (rather than another workspace) gets muddled: `<dest workspace>/methodconfigs/<source mconf>/copy`. I see a few solutions:

1. `<dest workspace>/methodconfigs/<dest mconf>/copy` with a reference to the mconf in the method repo in the payload (i.e. the code in this pull request);
2. `/methodconfigs/<source mconf>/copy` with the destination workspace and mconf names in the payload;
3. `<dest workspace>/methodconfigs/<source mconf>/copy` is terribly muddled and we shouldn't do it.

My vote is for #2; if we want to support copying mconfs between workspaces we can then do the (obvious) `<src workspace>/methodconfigs/<src mconf>/copy`.